File: server/src/main/java/net/bytemc/minestom/server/inventory/item/ClickType.java
package net.bytemc.minestom.server.inventory.item;

public enum ClickType {
    ALL("ALL"),
    LEFT("LEFT_CLICK"),
    RIGHT("RIGHT_CLICK"),
    SHIFT("START_SHIFT_CLICK"),
    DOUBLE("START_DOUBLE_CLICK");

    private final String identifier;

    ClickType(String identifier) {
        this.identifier = identifier;
    }

    public String getIdentifier() {
        return identifier;
    }
}

File: server/src/main/java/net/bytemc/minestom/server/inventory/item/ClickType.kt
package net.bytemc.minestom.server.inventory.item

enum class ClickType(var identifier: String) {
    ALL("ALL"),
    LEFT("LEFT_CLICK"),
    RIGHT("RIGHT_CLICK"),
    SHIFT("START_SHIFT_CLICK"),
    DOUBLE("START_DOUBLE_CLICK"),

}